examples/ktor/src/main/kotlin/com/expediagroup/graphql/examples/ktor/schema/HelloQueryService.kt

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2020 Expedia, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.graphql.examples.ktor.schema
+
+class HelloQueryService {
```

Comment:
If the route is deleted we can delete the service too

Reply:
This service is part of the graphql schema per spark example. It is convenient to have it to make a quick query.

Reply:
Ah I see, I thought this was for the hello http route